pyproject.toml

```diff
@@ -1,34 +1,260 @@
-[build-system]
-requires = [
-    "setuptools>=41.2",
-    "setuptools_scm[toml]>=1.15",
-    "wheel>=0.31.0",
+[tool.poetry]
+name = "openscm-runner"
+keywords = ["openscm", "runner", "python", "simple", "climate", "model"]
+version = "0.12.1"
+description = "Thin wrapper to run emissions scenarios with simple climate models"
+authors = [
+    "Zebedee Nicholls <zebedee.nicholls@climate-energy-college.org>",
+    "Robert Gieseke <rob.g@web.de>",
+    "Jared Lewis <jared.lewis@climate-resource.com>",
+    "Marit Sandstad <marit.sandstad@cicero.oslo.no>",
+    "Chris Smith <c.j.smith1@leeds.ac.uk>",
+    "Sven Willner <sven.willner@pik-potsdam.de>",
+]
+license = "BSD-3-Clause"
+readme = "README.md"
+repository = "https://github.com/openscm/openscm-runner"
+packages = [{include = "openscm_runner", from = "src"}]
+include = [
+    "adapters/fair_adapter/*.csv",
+    "adapters/ciceroscm_adapter/utils_templates/*.txt",
+    "adapters/ciceroscm_adapter/utils_templates/pam_RCMIP_test_klimsensdefault.scm",
+    "adapters/ciceroscm_adapter/utils_templates/run_dir/*.txt",
+    "adapters/ciceroscm_adapter/utils_templates/run_dir/scm_vCH4fb",
+    "adapters/ciceroscm_adapter/utils_templates/run_dir/input_OTHER/NATEMIS/*.txt",
+    "adapters/ciceroscm_adapter/utils_templates/run_dir/input_RF/RFLUC/*.txt",
+    "adapters/ciceroscm_adapter/utils_templates/run_dir/input_RF/RFSUN/*.txt",
+    "adapters/ciceroscm_adapter/utils_templates/run_dir/input_RF/RFVOLC/*.txt",
+    "adapters/ciceroscm_py_adapter/*.txt",
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "License :: OSI Approved :: BSD License",
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+    "Intended Audience :: Science/Research",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Scientific/Engineering",
 ]
-build-backend = "setuptools.build_meta"
 
-[tool.setuptools_scm]
+[tool.poetry.dependencies]
+python = "^3.9"
+openscm-units = ">=0.5.0"
+python-dotenv = "^1.0.0"
+scmdata = ">=0.7.4"
+tqdm = "^4.66.1"
+# optional below here
+ciceroscm = { version = "1.0.0", optional = true }  # TODO: ask Marit if she wants to loosen this
+fair = { version = "<2", optional = true }
+pymagicc = { version = "<3", optional = true }
+# Bit hacky, but basically we only need to pin pandas
```

Comment:
Is this because `ciceroscm` v1.0.0 doesn't pin pandas <2?

Reply:
Yep, Marit is going to follow up once I'm finished here